containers/server-image/Dockerfile

```
@@ -73,16 +78,23 @@ COPY java_agent.yaml /etc/prometheus-jmx_exporter/tomcat/java_agent.yml
 COPY java_agent.yaml /etc/prometheus-jmx_exporter/taskomatic/java_agent.yml
 COPY tomcat_jmx.conf /usr/lib/systemd/system/tomcat.service.d/jmx.conf
 COPY taskomatic_jmx.conf /usr/lib/systemd/system/taskomatic.service.d/jmx.conf
+COPY pam_susemanager /etc/pam.d/susemanager
 
 RUN chmod -R 755 /usr/bin/timezone_alignment.sh
 
 # Set a warning message for interactive shells
 COPY motd /etc/motd
 RUN echo "cat /etc/motd" >/etc/sh.shrc.local
 
+RUN pam-config -a --sss
+RUN pam-config -a --ldap
+RUN authselect select sssd --force
+RUN authselect apply-changes
```

Shouldn't those `RUN`s be chained into one? IIRC this reduces the number of image layers.

```suggestion
RUN pam-config -a --sss && \
         pam-config -a --ldap && \
         authselect select sssd --force && \
         authselect apply-changes
```